rclcpp/include/rclcpp/executor.hpp

```diff
@@ -280,6 +289,17 @@ class Executor
   void
   execute_any_executable(AnyExecutable & any_exec);
 
+  /**
+   * After executing an executable, this function determines
+   * if it should wake the wait in rcl_wait so that the executor
+   * can process any pending executable
+   *
+   * \return wake_after_execute_ flag
+   */
```

Comment:
Please update this one to have a brief using `///` as well.

Also the name of this function seems off. I would expect it to change the value of this variable rather than just return it (get it).